keps/sig-auth/1205-bound-service-account-tokens/README.md

```diff
@@ -343,35 +343,86 @@ are properly reloading tokens by:
+
+| Alpha | Beta | GA |
+| --- | --- | --- |
+| 1.10 | 1.12 | 1.20 |
+
+##### Alpha->Beta
+
+##### Beta->GA
+
+- [x] In use by multiple distributions
+- [x] Approved by PRR and scalability
+- [x] Known bugs fixed
+- [x] Tests passing
+  - [x] E2E test [ServiceAccounts should mount projected service account token when requested](https://k8s-testgrid.appspot.com/sig-auth-gce#gce)
+  - [x] E2E test [ServiceAccounts should set ownership and permission when RunAsUser or FsGroup is present](https://k8s-testgrid.appspot.com/sig-auth-gce#gce)
+  - [x] E2E test [ServiceAccounts should support InClusterConfig with token rotation](https://k8s-testgrid.appspot.com/sig-auth-gce#gce-slow)
+
+#### RootCAConfigMap
+
+| Alpha | Beta | GA |
+| --- | --- | --- |
+| 1.13 | 1.20 | 1.21 |
+
+#### Beta->GA
+
+- [ ] In use by multiple distributions
+- [ ] Approved by PRR and scalability
+- [ ] Known bugs fixed
 
-Estimated version: v1.20
+#### BoundServiceAccountTokenVolume
 
-All known migration frictions have been fixed:
+| Alpha | Beta | GA |
+| --- | --- | --- |
+| 1.13 | 1.21 | 1.22 |
 
-- PodSecurityPolicies that allow secrets but not projected volumes will
-  prevent the use of token volumes.
-  - Fixed in https://github.com/kubernetes/kubernetes/pull/92006
-- In-cluster clients that don’t reload service account tokens will start
-  failing an hour after deployment.
-  - Mitigation added in https://github.com/kubernetes/kubernetes/issues/68164
-- Pods running as non root may not access the service account token.
-  - Fixed in https://github.com/kubernetes/kubernetes/pull/89193
+##### Alpha->Beta
 
-An upgrade test is passing periodically:
+- [x] Known bugs fixed
+  - [x] PodSecurityPolicies that allow secrets but not projected volumes will
+    prevent the use of token volumes.
+    - Fixed in https://github.com/kubernetes/kubernetes/pull/92006
+  - [x] In-cluster clients that don’t reload service account tokens will start
+    failing an hour after deployment.
+    - Mitigation added in https://github.com/kubernetes/kubernetes/issues/68164
+  - [x] Pods running as non root may not access the service account token.
+    - Fixed in https://github.com/kubernetes/kubernetes/pull/89193
 
-1. Create pod A with feature disabled where pod A is working and a secret volume
-   is mounted.
-2. Enable feature where pod A continue working
-3. Create pod B and it is working and projected volumes are mounted.
+- [x] Tests passing
+  - [x] Upgrade test [sig-auth-serviceaccount-admission-controller-migration](https://k8s-testgrid.appspot.com/sig-auth-gce#upgrade-tests)
 
-#### Beta -> GA Graduation
+- [ ] RootCAConfigMap GA
 
-Estimated version: v1.21+
+##### Beta -> GA Graduation
 
-New `ServiceAccount` admission controller WAI in Beta for >= 1 minor without
+- [ ] Allow multi issuers to enable non distruptive issuer change.
+- [ ] New `ServiceAccount` admission controller WAI in Beta for >= 1 minor without
 significant issues.
 
 ## Production Readiness Review Questionnaire
```

Comment:
change "pods' service account tokens will not be long-lived" to "pods' service account tokens will expire after 1 year by default"

Reply:
drop the "pods created while the feature was enabled will reference a configmap that can grow stale with the feature disabled." bit from the PRR, since we're separating and prereqing the configmap controller feature gate so a single-version rollback will still keep the configmap current

Reply:
done